extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -471,6 +471,7 @@ public BytecodeRecorderConstantDefinitionBuildItem pregenProxies(
             LiveReloadBuildItem liveReloadBuildItem,
             ExecutorService buildExecutor) throws ExecutionException, InterruptedException {
         Set<String> managedClassAndPackageNames = new HashSet<>(jpaModel.getEntityClassNames());
+        Set<String> managedClassesName = new HashSet<>(jpaModel.getEntityClassNames());
```

Comment:
Let's make it clear that this set contains only `@Entity`-class names, managed classes could also be embeddables or mapped-superclasses.

```suggestion
        Set<String> entityClassesName = new HashSet<>(jpaModel.getEntityClassNames());
```